.eslintrc.js

```diff
@@ -24,16 +24,14 @@ module.exports = {
     {
```

Comment:
could you in the future move unrelated things to separate PRs? Makes it hard for me to review what you’ve asked me for

Reply:
OK, sorry about that.